lms/djangoapps/course_api/api.py

```diff
@@ -266,3 +269,89 @@ def get_course_run_url(request, course_id):
     """
     course_run_url = reverse('openedx.course_experience.course_home', args=[course_id])
     return request.build_absolute_uri(course_run_url)
+
+
+def get_course_members(course_key, page=1, limit=20, include_students=True, access_roles=None):
+    """
+    Returns a User queryset that filters all users related to a course. For example - Students, Teachers, Staffs etc.
+
+    Examples:
+        - Get all members:
+            get_course_member_queryset(course_key)
+        - Get only students excluding staffs:
+            get_course_member_queryset(course_key, include_students=True, access_roles=[])
+        - Get only staffs excluding students:
+            get_course_member_queryset(course_key, include_students=False, access_roles=None)
+        - Get only instructors:
+            get_course_member_queryset(course_key, include_students=False, access_roles=['instructor'])
+        - Get instructors & students:
+            get_course_member_queryset(course_key, include_students=True, access_roles=['instructor'])
+        - Get all members and prefetch course related roles:
+            get_course_member_queryset(course_key, prefetch_user_course_roles=True)
+
+    Arguments:
+        course_key (CourseKey): the CourseKey for the course
+        include_students: Wether or not to include students,
+        access_roles:
+            accepts an array of string course access roles.
+            If None provided, it includes all roles.
+        prefetch_user_course_roles:
+            prefetches CourseAccessRole & CourseEnrollment instances attached with user.
+            This only includes CourseAccessRole & CourseEnrollment instances related to
+            the provided CourseKey.
+
+    Returns:
+        (QuerySet): User queryset
+    """
+    queryset = User.objects.filter()
```

Comment:
Can you add a ordering query here? Otherwise the paginator might yield inconsistent results.
I got this warning on my devstack: 
```
2021-03-17 14:53:06,478 WARNING 300 [py.warnings] [user None] [ip None] warnings.py:109 - /edx/app/edxapp/edx-platform/lms/djangoapps/course_api/api.py:348: UnorderedObjectListWarning: Pagination may yield inconsistent results with an unordered object_list: <class 'django.contrib.auth.models.User'> QuerySet.
```

Reply:
Added order by user id.